packages/cli/src/tests/utils/astFixtures.js

```diff
@@ -58,6 +58,10 @@ const beforeRequest = (request, z, bundle) => {
 };
 
 const afterResponse = (response, z, bundle) => {
+  if (response.request.isShortHand === false) {
+    response.skipThrowForStatus = true;
+  }
+
   return z.legacyScripting.afterResponse(response, z, bundle);
```

Comment:
Updated legacy `afterResponse` snippets to be what https://github.com/zapier/zapier/pull/39898 changes them to.